graphql/schema.graphql

```diff
@@ -168,6 +168,16 @@ type Patch {
   baseVersionID: String
   moduleCodeChanges: [ModuleCodeChange!]!
   project: PatchProject
+  builds: [Build!]!
+}
+
+type Build {
+  id: String!
+  buildVariant: String!
+  status: String!
+  timeTaken: Duration!
+  predictedMakespan: Duration!
+  actualMakespan: Duration!
```

Comment:
Either this or timeTaken would be redundant wouldn't they? Which field are you planning on using in the UI?

Reply:
I will use actualMakespan and remove timeTaken